ci/prow/periodic_config.go

```diff
@@ -398,3 +402,38 @@ func generateIssueTrackerPeriodicJob(jobName, labelFilter, updatedTime, comment
 	addVolumeToJob(&data.Base, "/etc/housekeeping-github-token", "housekeeping-github-token", true, "")
 	executeJobTemplate(jobName, readTemplate(periodicCustomJob), "presubmits", "", data.PeriodicJobName, false, data)
 }
+
+// generateServingClusterUpdatePeriodicJobs generates periodic jobs to update serving clusters
+// that run performance testing benchmarks
+func generateServingClusterUpdatePeriodicJobs() {
+	recreateServingClustersJobName := "ci-knative-serving-recreate-clusters"
```

Comment:
You're not reusing the strings, why declare them and then have a single usage instead of embedding into the function call?

Reply:
Fixed.